pyempaq/common.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Facundo Batista
+# Licensed under the GPL v3 License
+# For further info, check https://github.com/facundobatista/pyempaq
+
+"""Common functionality."""
```

Comment:
"Common" between what parts of the system?

Reply:
Acá me imaginaba ir agregando funciones comunes a todo el "sistema". De ahí que no aclaré nada más. 
Entiendo que la recomendación es ser más explícito y transparente. Le pasé: "Common functionality for packer and unpucker modules"